app/src/main/res/values-ru/strings.xml

```diff
@@ -356,4 +360,7 @@
     <string name="own_location_size_summary" >Установите размер символа Вашего месторасположения</string>
     <string name="keep_zoom_goto_own_location_summary" >Установка сохранения Вашего текущего масштаба при центрировании собственного местоположения</string>
     <string name="keep_zoom_goto_own_location_title" >Сохранить масштаб</string>
+    <string name="notification_channel_name" translatable="false">Storm Alarms</string>
```

Comment:
Same here, I'm not sure if the channel name appears anywhere in the notifications.